gprotorch/gp.py

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
Stupid question but is this the one that wraps GraKel or the one that takes non-tensor inputs?

Reply:
So, SIGP just wraps non tensor inputs and external kernels can just be used by subclassing `Kernel` and defining the `kern` function.